packages/ckeditor5-editor-balloon/src/ballooneditor.js

```diff
@@ -17,7 +17,7 @@ import BalloonEditorUI from './ballooneditorui';
 import BalloonEditorUIView from './ballooneditoruiview';
 
 /**
- * The {@glink builds/guides/predefined-builds/overview#balloon-editor balloon editor} implementation (Medium-like editor).
+ * The {@glink /installation/advanced/predefined-builds#balloon-editor balloon editor} implementation (Medium-like editor).
```

Comment:
<img width="586" alt="image" src="https://user-images.githubusercontent.com/2270764/155734646-01e771c8-068c-42aa-83fa-85c349c2c4c2.png">

Please, review all absolute URLs.

Reply:
Addressed in [#11422](https://github.com/ckeditor/ckeditor5/pull/11422)